build.sbt

```diff
@@ -10,21 +10,17 @@ scmInfo in ThisBuild := Some(ScmInfo(
   url("https://github.com/slamdata/sbt-quasar-datasource"),
   "scm:git@github.com:slamdata/sbt-quasar-datasource.git"))
 
-// Include to also publish a project's tests
-lazy val publishTestsSettings = Seq(
-  publishArtifact in (Test, packageBin) := true)
-
 lazy val root = project
   .in(file("."))
-  .settings(noPublishSettings)
-  .aggregate(core)
-  .enablePlugins(AutomateHeaderPlugin)
-
-lazy val core = project
-  .in(file("core"))
-  .settings(name := "sbt-quasar-datasource")
   .settings(
-    /*
-    libraryDependencies += ...
-     */)
+    name := "sbt-quasar-datasource",
+    sbtPlugin := true,
+    sbtVersion := "1.1.6")
+  .settings(libraryDependencies ++= Seq(
+    "io.circe" %% "circe-core" % "0.10.1",
+    "io.get-coursier" %% "coursier" % "1.1.0-M7",
+    "io.get-coursier" %% "coursier-cache" % "1.1.0-M7",
+    // FIXME{coursier}: Align all versions once sbt-coursier-1.1.0-M8 is released
+    "io.get-coursier" %% "coursier-cats-interop" % "1.1.0-M8"
```

Comment:
This is a bit sketchy for now, but it works as the interop package doesn't depend on anything but the typeclass interfaces it implements, which were unchanged from `M7`.

Once a version of `sbt-coursier` is released for `M8` we can depend on that version for all the other libs.